### PR TITLE
Update readme files for the core libraries.

### PR DESCRIPTION
**Pull request by pallavit**

### CODE REVIEW

**Comment by pallavit** [comment]
Soft ping. Review please

**Review by srnagar — APPROVED** [review]

File: sdk/core/azure-core-amqp/README.md [hunk 1]
```diff
@@ -10,6 +10,38 @@ own AMQP client library that abstracts from the underlying transport library's i
 - A [Java Development Kit (JDK)][jdk_link], version 8 or later.
 
 ### Include the package
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```
Comment (joshfree) [inline sdk/core/azure-core-amqp/README.md:15]: > Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.

File: sdk/core/azure-core-http-netty/README.md [hunk 2]
```diff
@@ -9,6 +9,38 @@ Azure Core Netty HTTP client is a plugin for the `azure-core` HTTP client API.
 - A [Java Development Kit (JDK)][jdk_link], version 8 or later.
 
 ### Include the package
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```
Comment (joshfree) [inline sdk/core/azure-core-http-netty/README.md:14]: > Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.

File: sdk/core/azure-core-http-okhttp/README.md [hunk 3]
```diff
@@ -9,6 +9,38 @@ Azure Core OkHttp HTTP client is a plugin for the `azure-core` HTTP client API.
 - A [Java Development Kit (JDK)][jdk_link], version 8 or later.
 
 ### Include the package
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```
Comment (joshfree) [inline sdk/core/azure-core-http-okhttp/README.md:14]: > Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.

File: sdk/core/azure-core-serializer-json-gson/README.md [hunk 4]
```diff
@@ -9,6 +9,38 @@ Azure Core Gson JSON Serializer is a plugin for the `azure-core` `JsonSerializer
 - A [Java Development Kit (JDK)][jdk_link], version 8 or later.
 
 ### Include the package
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```
Comment (joshfree) [inline sdk/core/azure-core-serializer-json-gson/README.md:14]: > Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.

File: sdk/core/azure-core-serializer-json-jackson/README.md [hunk 5]
```diff
@@ -9,6 +9,38 @@ Azure Core Jackson JSON Serializer is a plugin for the `azure-core` `JsonSeriali
 - A [Java Development Kit (JDK)][jdk_link], version 8 or later.
 
 ### Include the package
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```
Comment (joshfree) [inline sdk/core/azure-core-serializer-json-jackson/README.md:14]: > Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.

File: sdk/core/azure-core/README.md [hunk 6]
```diff
@@ -19,9 +19,40 @@ APIs in one client library, you will know how to use them in other client librar
 
 ### Include the package
 
-Typically, you won't need to install or depend on Azure Core, instead it will be transitively downloaded by your build
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```
Comment (joshfree) [inline sdk/core/azure-core/README.md:24]: > Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.

**Review by joshfree — APPROVED** [review]
LGTM

**Comment by pallavit** [comment]
Thank you for the review.

**Pull request merged**